WordPress/src/main/java/org/wordpress/android/ui/bloggingreminders/BloggingRemindersViewModel.kt

```diff
@@ -76,7 +79,8 @@ class BloggingRemindersViewModel @Inject constructor(
         }.asLiveData(mainDispatcher)
     }
 
-    fun showBottomSheet(siteId: Int, screen: Screen) {
+    fun showBottomSheet(siteId: Int, screen: Screen, fromSiteSettings: Boolean) {
+        isFromSiteSettings = fromSiteSettings
```

Comment:
I'd suggest adding this parameter to the `Screen`. We can either create a new enum value or we could have a sealed class instead and have the `Prologue` parametrized.

Reply:
I've added another item to Screen enum to cater for starting from settings. thanks